index.html

```diff
@@ -0,0 +1,268 @@
+        <p class="features__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+        </p>
+      </div>
+      <div class="features__card-items  ">
+        <img class="features__icon" src="./img/Icon@3x.svg" alt="">
+        <h2 class=" features__title features__title--medium">Advanced 256-bit encryption</h2>
+        <p class="features__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+        </p>
+      </div>
+    </div>
+
+
+  </section>
+  <!-- clients -->
+  <section class="clients">
+    <h2 class="clients__title">The stunning results our <br>customers have experienced</h2>
+    <div class="clients__team">
+      <div class="clients__card">
+        <p class="clients__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+          eirmod tempor invidunt ut labore et doloremagna aliquyam erat, sed diam voluptua.</p>
+        <div class="clients__details">
+          <img class="clients__image" src="./img/photography-of-a-guy-wearing-green-shirt-1222271.svg" alt="">
+          <h2 class="clients__name">Ronwood</h2>
+          <p class=" clients__position">CEO</p>
+        </div>
+      </div>
+
+      <div class="clients__card">
+        <p class="clients__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+          eirmod tempor invidunt ut labore et doloremagna aliquyam erat, sed diam voluptua.</p>
+        <div class="clients__details">
+          <img class="clients__image" src="./img/photography-of-a-guy-wearing-green-shirt-1222271.svg" alt="">
+          <h2 class="clients__name">Ronwood</h2>
+          <p class=" clients__position">CEO</p>
+        </div>
+      </div>
+
+      <div class="clients__card">
+        <p class="clients__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+          eirmod tempor invidunt ut labore et doloremagna aliquyam erat, sed diam voluptua.</p>
+        <div class="clients__details">
+          <img class="clients__image" src="./img/photography-of-a-guy-wearing-green-shirt-1222271.svg" alt="">
+          <h2 class="clients__name">Ronwood</h2>
+          <p class=" clients__position">CEO</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- footer -->
+  <section class="footer">
+    <div class="footer__container ">
+      <h2 class="footer__title">Are you ready to grow your<br> business with us?</h2>
+      <p class="footer__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+      <button class="footer__button footer__button--small">Get free trial</button>
+    </div>
+    <div class="footer__content">
+      <div class="footer__cols">
+        <ul class="footer__link-container">
+          <li class="footer__link-title">Company</li>
+          <li class="footer__link"><a href="#">About us</a></li>
```

Comment:
this `a` tag should have a class and try to format you code that it would be visble